pages/22.What's New/docs.en.md

```diff
@@ -13,6 +13,32 @@ taxonomy:
 ----------------------------
 In this section we'll highlight the new features for each major release of Mautic.  They are in version order beginning with the latest.
 
+##Version 3.0
```

Comment:
As applications [don't always handle a missing space correctly](https://www.markdownguide.org/basic-syntax/#heading-best-practices), let's make sure we leave one just in case.